### ISSUE TITLE
version

### ISSUE THREAD

**Issue by henrikstranneheim** [J35P312/TIDDIT#1]
I installed 1.1.0, but the version seems to not have changed!

```
$TIDDIT
TIDDIT-1.0.5 - a structural variant caller
Usage: TIDDIT <module> [options]
modules
	--help	produce help message
	--sv	select the sv module to find structural variations
	--cov	select the cov module to analyse the coverage of the genome using bins of a specified size```

**Comment by J35P312** [comment]
woopsie daisy! Sorry about that, I have generated a new release with a proper sample tag

**Comment by henrikstranneheim** [comment]
I hate to tell you this, but you forgot to update the version flag again for the new release

**Comment by J35P312** [comment]
Thanks! Now it is fixed(version 1.1.3), There really should be somekind of system for syncing those flags =P
